plugins/ShapeLibrary/README.md

```diff
@@ -0,0 +1,44 @@
+# Quick Shape Library
+
+Quick Shape Library for inserting predefined shapes into a Xournal++ document via this plugin along with Extract Stroke Info for creating new shapes.
+
+## Installation Steps
+
+1. **Place the folder** (`ShapeLibrary`) in:
+   - `C:\Users\<user>\AppData\Local\xournalpp\plugins\` on Windows 
+     *Note: The `AppData` folder may be hidden.*
+   - `~/.config/xournalpp/plugins` on Linux or MacOS
+
+1. **Place the icons** (`shapes_symbolic.svg` and `extract-info-symbolic.svg`) in:
+   - `C:\Users\<user>\AppData\Local\share\icons\hicolor\scalable\actions` on Windows,
+   - `~/.local/share/icons/hicolor/scalable/actions/` on Linux or MacOS
+
+2. **In the Xournal++ app**:
+  Open menu `Edit > Toolbars > Customize`. You will find the copied icons in the `Plugins` section. Place them at a suitable location in the toolbar.
+
+3. **Activate the plugin** in menu `Plugins > Plugin Manager`
+
+4. **Use the plugin** as needed
+
+
+## Add Your Own Shape with "Extract_Stroke_Info"
+
+1. **Draw your own shape** in Xournal++ and place it at the top left corner of the page.
+  *Note: The shape will be inserted at the same position. When you want to insert the sape if the page is smaller than the current one, you may not be able to see the inserted shapes.*
```

Comment:
Well we could also do this in the plugin (placing the icon at the top left corner by translating the coordinates) and then on insertion e.g. place the icon at the center of the page (or viewport if this is possible).

A bit sad that we currently don't have a way of getting something like pointer coordinates for plugins. This would then be going into the direction of making custom tools via plugins. But this is way outside of the scope of this plugin (maybe just something to keep in mind for the future).
Maybe related: https://github.com/xournalpp/xournalpp/pull/4703